Db/src/main/java/org/gusdb/fgputil/db/runner/SQLRunner.java

```diff
@@ -195,6 +201,29 @@ public SQLRunner(Connection conn, String sql, String sqlName) {
     _sqlName = sqlName;
   }
 
+  /**
+   * Set a global connection timeout that gets used for all SQLRunner instances if no override is set. Set to null
+   * if no global default timeout is desired.
+   *
+   * @param timeout Default timeout value.
+   */
+  public static void setGlobalDefaultConnectionTimeout(Duration timeout) {
+    GLOBAL_QUERY_TIMEOUT_SECONDS = Optional.ofNullable(timeout)
+        .map(d -> (int) d.getSeconds())
+        .orElse(null);
+  }
+
+  /**
+   * Set the override connection timeout for this instance of SQLRunner. This takes precedence of the global timeout.
+   *
+   * @param timeout Timeout value to propagate to JDBC statement.
+   * @return this instance of SQLRunner.
+   */
+  public SQLRunner setDefaultConnectionTimeout(Duration timeout) {
```

Comment:
Naming- How about setConnectionTimeoutOverride() to sync with the documentation above "if no override is set".  This isn't a "default" since it's the one that will be used if set.